x-pack/plugins/secops/public/components/resize_handle/index.test.tsx

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import 'jest-styled-components';
+import * as React from 'react';
+
+import { TestProviders } from '../../mock/test_providers';
+
+import {
+  addGlobalResizeCursorStyleToBody,
+  globalResizeCursorClassName,
+  isResizing,
+  removeGlobalResizeCursorStyleFromBody,
+  Resizeable,
+} from '.';
+import { CellResizeHandle } from './cell_resize_handle';
+
+describe('Resizeable', () => {
+  beforeEach(() => {
+    document.body.classList.remove(globalResizeCursorClassName);
+  });
+
+  afterEach(() => {
+    document.body.classList.remove(globalResizeCursorClassName);
+  });
```

Comment:
Interesting you need this in both the `beforeEach` and `afterEach`. I wouldn't expect you would need it in both but just in the `beforeEach`. No changes, it's just curious looking.